frameworks/D/vibed/source/app.d

```diff
@@ -0,0 +1,102 @@
+import vibe.appmain;
+import vibe.d;
+import std.random;
+
+const worldSize = 10000;
+const fortunesSize = 100;
+const mongoUrl = "mongodb://127.0.0.1/";
+
+MongoClient mongo;
+MongoCollection worldCollection;
+MongoCollection fortunesCollection;
+
+shared static this()
+{
+	mongo = connectMongoDB( mongoUrl );
+	worldCollection = mongo.getCollection( "hello_world.World" );
+	fortunesCollection = mongo.getCollection( "hello_world.Fortunes" );
+
+	auto router = new URLRouter;
+	router.get("/plaintext", &plaintext);
+	router.get("/json", &json);
+	router.get("/db", &db);
+	router.get("/queries", &queries);
+	router.get("/generate-world", &generateWorld);
+	router.get("/generate-fortunes", &generateFortunes);
+	router.get("/", staticTemplate!"index.dt");
+
+	auto settings = new HTTPServerSettings;

```

Comment:
No, I am simply getting it to the point where it works in the suite. After this gets merged in (soon), others can open pull request to fine-tune it.